default/src/components/asaTransfer.js

```diff
@@ -0,0 +1,69 @@
+/* global AlgoSigner */
+import algosdk from 'algosdk';
+
+import { networks } from '../../algob.config';
+import { LEDGER } from '../algosigner.config';
+
+const LAST_ROUND = 'last-round';
+const CONFIRMED_ROUND = 'confirmed-round';
+
+const algodClient = new algosdk.Algodv2(
+  networks.default.token,
+  networks.default.host,
+  networks.default.port
+);
+
+const waitForConfirmation = async function (txId) {
+  let response = await algodClient.status().do();
```

Comment:
Can't we use the `Algosigner.algod(..)` to do it?

Reply:
I think `AlgoSigner.algod` makes direct calls to the blockchain. Here, our requirement is easily satisfied using `algodClient`, with `AlgoSigner.algod` we would have to hardcode the exact path with `algodClient` is calling.